equinix/resource_fabric_cloud_router.go

```diff
@@ -15,47 +18,250 @@ import (
+			Optional:    true,
+			Description: "Customer-provided Fabric Cloud Router description",
+		},
+		"state": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Fabric Cloud Router overall state",
+		},
+		"equinix_asn": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Equinix ASN",
+		},
+		"package": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Fabric Cloud Router Package Type",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: FabricCloudRouterPackageSch(),
+			},
+		},
+		"change_log": {
+			Type:        schema.TypeSet,
+			Computed:    true,
+			Description: "Captures Fabric Cloud Router lifecycle change information",
+			Elem: &schema.Resource{
+				Schema: equinix_fabric_schema.ChangeLogSch(),
+			},
+		},
+		"type": {
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringInSlice([]string{"XF_ROUTER"}, true),
+			Description:  "Defines the FCR type like XF_ROUTER",
+		},
+		"location": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Fabric Cloud Router location",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: equinix_fabric_schema.LocationSch(),
+			},
+		},
+		"project": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Customer resource hierarchy project information.\nApplicable to customers onboarded to Equinix Identity and Access Management. For more information see Identity and Access Management: Projects",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: FabricCloudRouterProjectSch(),
+			},
+		},
+		"account": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Customer account information that is associated with this Fabric Cloud Router",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: FabricCloudRouterAccountSch(),
+			},
+		},
+		"order": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Order information related to this Fabric Cloud Router",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: equinix_fabric_schema.OrderSch(),
+			},
+		},
+		"notifications": {
+			Type:        schema.TypeList,
+			Required:    true,
+			Description: "Preferences for notifications on Fabric Cloud Router configuration or status changes",
+			Elem: &schema.Resource{
+				Schema: equinix_fabric_schema.NotificationSch(),
+			},
+		},
+		"bgp_ipv4_routes_count": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Access point used and maximum number of IPv4 BGP routes",
+		},
+		"bgp_ipv6_routes_count": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Access point used and maximum number of IPv6 BGP routes",
+		},
+		"distinct_ipv4_prefixes_count": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Number of IPv4 BGP routes in use (including non-distinct prefixes).",
```

Comment:
I think this description should be the above, and it seems they are switched. Look in the documentation link shared and update.